### PR TITLE
fix(@desktop/onboarding): the profile image doesn't look good on the onboarding page

### LINKED ISSUE #5241: The profile image doesn't look good on the onboarding page
# Bug Report

## Steps to reproduce

1. Launch the app
3. Hit "I'm new to Status" - "Generate new keys"
4. Hit "+" and upload the photo from the attachment to this issue
5. Hit "Done"
6. Check the profile picture
camera_test_apple_563728

#### Expected behavior
The profile picture is shown in full of the circle shape
#### Actual behavior
The profile picture is shown with white borders
image

### Additional Information

- Status desktop version: commit dcab50fe0944f65a1fafef3fdb9e7abe66277b6c
- Operating System: MacOS Big Sur 11.6.5 (20G526)

### PR DESCRIPTION
**Pull request by saledjenic**
The issue occurred cause image was not cropped properly, but used the original one. 

While I was working on this issue I noticed that we're using calls from some modules, like main module, profile module... that are available only after user logs in successfully, and that display some errors/warning in the log.

Corresponding `status-go` PR: 
- https://github.com/status-im/status-go/pull/2627

Corresponding `nim-status-go` PR: 
- https://github.com/status-im/nim-status-go/pull/12

Fixes #5241

### CODE REVIEW

**Review by abhi-status — APPROVED** [review]

**Review by endulab — CHANGES_REQUESTED** [review]
@saledjenic I found 1 issue:
1. Click on the cirlce
2. Choose a file
3. Do not crop - exit with X
4. Click on circle one more time and choose the same file
5. Nothing happens

File: ui/app/AppLayouts/Onboarding/popups/UploadProfilePicModal.qml [hunk 1]
```diff
@@ -28,16 +29,23 @@ ModalPopup {
     Item {
         anchors.fill: parent
 
-        RoundedImage {
+        StatusRoundedImage {
```
Comment (endulab) [inline ui/app/AppLayouts/Onboarding/popups/UploadProfilePicModal.qml:32]: You could even use StatusSmartIdenticon here.

**Comment by abhi-status** [comment]
> @saledjenic I found 1 issue:
> 
> 1. Click on the cirlce
> 2. Choose a file
> 3. Do not crop - exit with X
> 4. Click on circle one more time and choose the same file
> 5. Nothing happens

Thanks Michal!
I can see this issue as well.
Could you please check Sale ?

The issue is that I can select a pic and then go on to not choose it by clicking `x` on the cropping modal, but I cannot seem to select this same pic to crop again.
Clicking on other pics work as expected, until I hit the `x` on the cropping modal.

File: ui/app/AppLayouts/Onboarding/popups/UploadProfilePicModal.qml (hunk 1 above)
Comment (saledjenic) [reply ui/app/AppLayouts/Onboarding/popups/UploadProfilePicModal.qml:33]: @endulab yes, but the thing is that under the hood `StatusSmartIdenticon` is again `StatusRoundedImage` and if I used `StatusSmartIdenticon` I wouldn't be able to set transparent background and border for the image at the same time, cause they depends on `isIdenticon` prop, and if I set it to true then I would get black background , if I set it to false then I would not have border.

**Comment by saledjenic** [comment]
> Could you please check Sale ?

Checked, fixed.

**Review by richard-ramos — APPROVED** [review]

**Review by osmaczko — APPROVED** [review]

**Review by endulab — APPROVED** [review]

**Review by elina2015 — APPROVED** [review]
Tested in https://ethstatus.testrail.net/index.php?/runs/view/8992&group_by=tests:status_id&group_order=asc

**Comment by saledjenic** [comment]
@abhi-status @elina2015 I would like to merge this, but I see label set here is `tested` + `tested-issues`, did you forget to remove `tested-issues` label or?

**Comment by abhi-status** [comment]
@saledjenic yes, that was meant to be removed after testing.

**Pull request merged**